src/services/redisSessionStorage.ts

```diff
@@ -83,7 +83,16 @@ export default class RedisSessionStorage extends RedisStorageUtils
       user.fiscal_code
     );
 
-    const removeOtherUserSessionsPromise = this.removeOtherUserSessions(user);
+    // Read ENV to allow multiple user's sessions functionality
+    // Default value is false when the ENV var is not provided
+    const ALLOW_MULTIPLE_SESSIONS = fromNullable(
+      process.env.ALLOW_MULTIPLE_SESSIONS
+    )
+      .map(_ => Boolean(_))
```

Comment:
Is this a string ? I'm asking as `Boolean("false")` will result in a true value when for example `ALLOW_MULTIPLE_SESSIONS=false`

Reply:
Sorry, it's a mine big mistake.
Fixed in 67edc15